python/cuml/svm/svm_base.pyx

```diff
@@ -198,6 +198,9 @@ class SVMBase(Base):
         self.epsilon = epsilon
         self.svmType = None  # Child class should set self.svmType
 
+        # Parameter to indicating if model has been fitted at least once
```

Comment:
Note that Sklearn uses fit_status the following way: 
- 0 means correctly fitted
- 1 otherwise, which can also mean that the model is fitted but it did not converge up to the required precision.
- The variable is not defined before we fit the model

Ideally, in fit we should check whether we are converged and set the flag accordingly. I just filed an issue for that #1631. Here I would not hijack value 1 to indicate not yet fitted. One could use -1, or `hasattr(self, 'fit_status_')`: 
```suggestion
        # Parameter to indicate if model has been correctly fitted
        # fit_status == -1 indicates that the model is not yet fitted
        self.fit_status_ = -1
```

Reply:
That makes sense, I'll make those changes. Thank you! :+1: